### PR TITLE
fix: ColorTokenTable sticky content switcher

### LINKED ISSUE #1302: Style regression: top of color usage page messed up
Looks like the top of the color usage page has been messed up.

**What its looks like:**

image

**What is should-ish look like** (only comp I could find without Jeannie, but basically correct).

image

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #1302 

Fixes the sticky content switcher on the Guidelines > Color > Usage page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,8 @@
   "dependencies": {
     "carbon-components-react": "^7.18.0",
     "classnames": "^2.2.6",
-    "react": "^16.8.6"
+    "react": "^16.8.6",
+    "react-copy-to-clipboard": "^5.0.1"
```
Comment (jnm2377) [inline package.json:47]: fyi, this change was bc of an eslint error saying it should be a dependency not a devDependency

**Review by jeanservaas — COMMENTED** [review]
This looks awesome! 

Just one tweak, could you adjust the space above the H3 from 80px to 48px.

image

**Review by vpicone — APPROVED** [review]

**Review by aagonzales — APPROVED** [review]
🎉

**Comment by joshblack** [comment]
Hey! Just a quick reminder to stick to the assigned codeowner reviewers for PRs 👀   Totally fair to add in additional reviewers but it's important that we keep the assigned codeowners in there as well, really just so the branch isn't blocked from merging because of the codeowners requirement 👍

File: src/components/ColorTokenTable/ColorTokenTable.js [hunk 2]
```diff
@@ -45,7 +46,8 @@ export default class ColorTokenTable extends React.Component {
 
   addScrollListener() {
     document.addEventListener('scroll', () => {
-      const stickyPoint = this.state.mobile ? 400 : 398;
+      console.log(window.scrollY);
```
Comment (joshblack) [inline src/components/ColorTokenTable/ColorTokenTable.js:49]:
```suggestion
```

File: src/components/ColorTokenTable/color-token-table.scss [hunk 3]
```diff
@@ -6,9 +6,18 @@
 .color-token-table__theme-switcher {
   position: relative;
   width: 100%;
-  height: rem(48px);
+  height: 3rem;
   margin-bottom: 1.75rem;
   background: $ui-02;
+
+  @media all and (max-width: 1055px) {
```
Comment (joshblack) [inline src/components/ColorTokenTable/color-token-table.scss:13]: Quick question, where are the breakpoint values coming from? Are they tied to a range of device sizes or a particular layout shift that's happening?

**Review by joshblack — APPROVED** [review]
Looks good! Just left a couple of comments/questions 👀

**Comment by jnm2377** [comment]
Hey @joshblack thanks for the review. I removed some of the reviewers since this issue was assigned to Aux by Mike (which probably wasn't the best thing to do considering System squad works on the site) and requested review from the Aux devs. I also requested review from Jeannie and Anna so I figured that their approval would allow us to merge bc they're part of system squad. I think going forward, we're gonna be talking to Mike to no longer assign us carbon-website issues since System squad is owning that. It just gets confusing trying to figure out what team is taking care of what.

File: src/components/ColorTokenTable/color-token-table.scss (hunk 3 above)
Comment (jnm2377) [reply src/components/ColorTokenTable/color-token-table.scss:13]: Hmmmm, I'm not sure. This was an existing style that I just moved up.

Comment (jnm2377) [reply src/components/ColorTokenTable/color-token-table.scss:13]: It appears that it is part of our breakpoints (Large = 1056px).

File: src/components/ColorTokenTable/ColorTokenTable.js (hunk 2 above)
Comment (jnm2377) [reply src/components/ColorTokenTable/ColorTokenTable.js]: good catch! 😄

**Comment by alisonjoseph** [comment]
+1 to @jnm2377 Mike has still been assigning issues to aux specifically causing confusion since codeowners tags system not aux, moving forward Carbon website issues should be brought up and prioritized in system squad sprints.

File: src/components/ColorTokenTable/color-token-table.scss (hunk 3 above)
Comment (jnm2377) [reply src/components/ColorTokenTable/color-token-table.scss:13]: I'll update it to use our mixin/breakpoint token

**Comment by joshblack** [comment]
@jnm2377 totally makes sense!

**Review by jeanservaas — APPROVED** [review]

**Pull request merged**